.storybook/main.js

```diff
@@ -11,23 +15,22 @@ const SOURCE_COMPONENTS_DIR = path.join(SOURCE_DIR, 'components');
 async function findStories() {
   return glob
     .sync('src/**/*.stories.@(jsx|mdx)')
-    .filter(storiesPath => !storiesPath.includes('.chromatic.stories.'))
-    .map(storiesPath => path.relative(__dirname, storiesPath));
+    .filter((storiesPath) => !storiesPath.includes('.chromatic.stories.'))
+    .map((storiesPath) => path.relative(__dirname, storiesPath));
 }
 
 module.exports = {
   stories:
-    process.env.CHROMATIC === 'true'
+    process.env.STORYBOOK_ENV === 'chromatic'
```

Comment:
we need to change env variable in github action intergate workflow

Reply:
Yes, forgot about this script, thanks :)

Reply:
Done